src/v2/Components/Artwork/Details.tsx

```diff
@@ -112,24 +127,22 @@ export class Details extends React.Component<Props, null> {
   }
 
   saleInfoLine() {
-    if (!this.props.showSaleLine) {
-      return null
-    }
+    const { useLighterFont } = this.props
 
     return (
       <>
         <Sans
           style={{ display: "inline" }}
-          color={color("black100")}
-          weight={"medium"}
-          size={"2"}
+          color={useLighterFont ? color("black60") : color("black100")}
```

Comment:
I don't _think_ the `color` helper is needed here, but not totally sure.

Reply:
It is because on the Top Works rail the design spec indicates the price metadata should be `black60` and on the artwork grids its `black100`